.circleci/config.yml

```diff
@@ -303,32 +303,32 @@ workflows:
           cuda_version: "9.2"
           # Excluding CudaGdr for lack of InfiniBand hardware, and CudaIpc on
           # multi GPU for lack of p2p capabilities.
-          exclude_tests: "CudaGdr*:CudaIpc/CudaMultiGPUChannelTestSuite*"
+          exclude_tests: "CudaGdr*:CudaIpc/CudaMultiGPUChannelTestSuite*:*CudaIpcCanCommunicateWithRemote*"
```

Comment:
Is there a way we could actually test IPC too? For example, if we set CUDA_VISIBLE_DEVICES=0 on both endpoints, they would see the same device and this device would support P2P onto itself, thus IPC should be viable.